src/main/java/com/blackducksoftware/integration/hub/alert/channel/SupportedChannels.java

```diff
@@ -22,15 +22,8 @@
  */
 package com.blackducksoftware.integration.hub.alert.channel;
 
```

Comment:
If we like this approach, this will eventually be an enum. I'm leaving them as strings for ease of refactoring right now.